notebooks/227-whisper-subtitles-generation/227-whisper-nncf-quantize.ipynb

```diff
@@ -0,0 +1,941 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    #!pip install -q evaluate
</pre><p> </p><p>I think this notebook is missing dependencies: <code>librosa, soundfile, jiwer</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1237/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
Thanks!